install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3295,6 +3295,8 @@
 
 if (isDatabaseVersion('201603150')) {
 
+	Settings::AddNew("panel.db_version", "201604120");
```

Comment:
I explicitly commented on the corresponding line in my last comment...that's not what i meant, you do not add "panel.db_version" as NEW setting....what i meant was you actual NEW SETTING which would be system.ssl_protocols, therefore please remove this line and exchange your INSERT INTO sql line below with

Settings::AddNew("system.ssl_protocols", "...");

Also, please check out "preconfig" stuff so the admin can set the initial value of this setting right from within the update, and not gets the hardcoded default